projects/components/src/datagrid/filters/datagrid-filter.ts

```diff
@@ -93,6 +93,8 @@ export abstract class DatagridFilter<V, C extends FilterConfig<V>> extends Subsc
         this.subscribe(obs, () => this.changes.next());
     }
 
+    ngOnDestroy(): void {}
```

Comment:
If we go down the route of making this abstract, ain't we supposed to call unsubscribeAll here?

Reply:
The logic of calling unsubscribeAll is handled by SubscriptionTracker in the constructor. It modifies the ngOnDestroy method itself.